lib/games.rb

```diff
@@ -101,6 +101,19 @@ def average_goals(stat)
     goals_collection
   end
 
+  def average_goals(stat)
+    goals_collection = {}
+    stat_collect(stat).each do |collection|
+      collection = collection.to_sym
+      goals_collection[collection] = 0
+      average = total_goals(stat)[collection] / stat_games_count(stat)[collection].to_f
+      goals_collection[collection] = average
+    end
+    goals_collection
+  end
```

Comment:
Take a look at this method. Pretty clean.

Reply:
It looks good! I've been thinking that using the term 'collection' for the elements is misleading though, can we revise 'collection' to 'element' instead? I'm open to other ideas for what to call it. At the point of reference in the block on line 106, we're referencing each single element in our collection.